projects/golang/go/Makefile

```diff
@@ -80,8 +80,15 @@ clean:
 	rm -rf $(HOME)/rpmbuild
 
 .PHONY: install-deps
+install-deps: T_GO_VERSION=1.15.15
+install-deps: T_GO_RELEASE_NUMBER=1
```

Comment:
any reason not to source this from the release trigger file for the given Go version?

Reply:
we'd need to account for the new prod release case but it'd be ideal if this wasn't hardcoded